ansible/aws_coreos_site.yml

```diff
@@ -105,6 +148,9 @@
           - protocol: http
             load_balancer_port: 80
             instance_port: 80
+          - protocol: http
+            load_balancer_port: 8080
```

Comment:
It can't, its mapping 80 to the registry, we can't proxy that through
Vulcan, well we can but its rather complex
On 5 Jan 2016 12:15 am, "Catalin Ilea" <notifications@github.com> wrote:

> In ansible/aws_coreos_site.yml
> <https://github.com/Financial-Times/coco-docker-image-cache-provisioner/pull/7#discussion_r48733697>
> :
>
> > @@ -105,6 +148,9 @@
> >            - protocol: http
> >              load_balancer_port: 80
> >              instance_port: 80
> > +          - protocol: http
> > +            load_balancer_port: 8080
>
> Anyway, minor detail...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Financial-Times/coco-docker-image-cache-provisioner/pull/7/files#r48733697>
> .
>